core/controllers/contributor_dashboard.py

```diff
@@ -882,6 +882,52 @@ def get(self) -> None:
         self.render_json(self.values)
 
 
+class TranslatableTopicNamesPerClassRoomHandlerDict(TypedDict):
+    """A dictionary representing all topics associated to classroom."""
+
+    classRoom: str
+    topics: List[str]
+
+
+class TranslatableTopicNamesPerClassRoomHandler(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+):
+    """Provides names of all translatable topics 
+    associated with classroom in the datastore."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+    HANDLER_ARGS_SCHEMAS: Dict[str, Dict[str, str]] = {'GET': {}}
+
+    @acl_decorators.open_access
+    def get(self) -> None:
+        """Gets all translatable topics grouped by classroom.
+        Returns a JSON response containing topics organized by classroom name.
+        """
+        # Build mapping of topic IDs to classroom names.
+        topic_id_to_classroom = {
```

Comment:
this dict only include the topic_ids with a classroom. how this function handles the topic without a classroom. (As the provided screenshot shows in the pr description.)